packages/icons/src/service-grafana-loki.svg

```diff
@@ -0,0 +1 @@
nt id="b" gradientUnits="userSpaceOnUse" x1="7.8816" x2="4.16199" xlink:href="#a" y1="25.8847" y2="1.71339"/><linearGradient id="c" gradientUnits="userSpaceOnUse" x1="18.005" x2="14.2854" xlink:href="#a" y1="24.3271" y2=".155872"/><linearGradient id="d" gradientUnits="userSpaceOnUse" x1="12.8316" x2="9.11201" xlink:href="#a" y1="25.123" y2=".951689"/><linearGradient id="e" gradientUnits="userSpaceOnUse" x1="10.3576" x2="6.638" xlink:href="#a" y1="25.5038" y2="1.33252"/><linearGradient id="f" gradientUnits="userSpaceOnUse" x1="7.8792" x2="4.15958" xlink:href="#a" y1="25.8851" y2="1.71383"/><linearGradient id="g" gradientUnits="userSpaceOnUse" x1="18.0071" x2="14.2875" xlink:href="#a" y1="24.3264" y2=".155139"/><linearGradient id="h" gradientUnits="userSpaceOnUse" x1="12.8315" x2="9.11181" xlink:href="#a" y1="25.1229" y2=".951607"/><linearGradient id="i" gradientUnits="userSpaceOnUse" x1="10.3575" x2="6.638" xlink:href="#a" y1="25.5036" y2="1.3325"/><linearGradient id="j" gradientUnits="userSpaceOnUse" x1="7.35156" x2="3.82384" xlink:href="#a" y1="25.4857" y2="2.56143"/><linearGradient id="k" gradientUnits="userSpaceOnUse" x1="8.36977" x2="4.58226" xlink:href="#a" y1="25.9797" y2="1.36704"/><linearGradient id="l" gradientUnits="userSpaceOnUse" x1="10.0201" x2="6.00428" xlink:href="#a" y1="26.2975" y2=".201179"/><linearGradient id="m" gradientUnits="userSpaceOnUse" x1="10.8045" x2="7.14337" xlink:href="#a" y1="25.2886" y2="1.49706"/><linearGradient id="n" gradientUnits="userSpaceOnUse" x1="12.199" x2="8.94671" xlink:href="#a" y1="24.0498" y2="2.91518"/><linearGradient id="o" gradientUnits="userSpaceOnUse" x1="13.1675" x2="9.79052" xlink:href="#a" y1="24.2131" y2="2.26852"/><clipPath id="p"><path d="m3.72217 2h17.1429v20h-17.1429z"/></clipPath><g clip-path="url(#p)"><path d="m8.02478 21.616-.23714-1.5923-1.59233.2372.27103 1.5923z" fill="url(#b)"/><path d="m20.4926 17.1441-.271-1.5584-6.9452 1.0841.2032 1.5585z" fill="url(#c)"/><path d="m11.0062 18.6347 1.5923-.2711-.2371-1.5584-1.5924.2371z" fill="url(#d)"/><path d="m10.5321 21.2096-.271-1.5585-1.55846.2372.20328 1.5923z" fill="url(#e)"/><path d="m5.82275 17.7877.23716 1.5923 1.59233-.2371-.23715-1.5924z" fill="url(#f)"/><path d="m13.6147 19.1088.2711 1.6263 6.9791-1.0842-.2372-1.5923z" fill="url(#g)"/><path d="m11.4126 21.0739 1.5585-.2033-.2372-1.6262-1.5923.271z" fill="url(#h)"/><path d="m8.2959 17.4149.271 1.5923 1.5585-.2371-.23722-1.5924z" fill="url(#i)"/><path d="m6.36474 16.8053-1.93112-12.67078-.71145.10163 1.96499 12.67085z" fill="url(#j)"/><path d="m7.27957 16.6697-2.10052-13.61941-.67759.13553 2.10052 13.58558z" fill="url(#k)"/><path d="m8.83824 16.4325-2.20216-14.4325-.67758.1355 2.20213 14.3648z" fill="url(#l)"/><path d="m9.75263 16.297-2.03275-13.14514-.64371.10163 2.03275 13.11121z" fill="url(#m)"/><path d="m11.3111 16.026-1.7956-11.65442-.67761.10163 1.79561 11.68839z" fill="url(#n)"/><path d="m12.226 15.8905-1.8634-12.09485-.67754.10164 1.89724 12.09491z" fill="url(#o)"/></g></svg>
```

Comment:
nit: Do we require fill="none" here? If not, we can remove it

Reply:
This also has linear gradient. Can we consider png instead?
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Reply:
@AshwinBhatkal-Harness I will remove `fill="none"` attribute. As the size of SVG is not exceeding 5K, shall we use SVG itself as suggested in the link?

Reply:
Yes. Works